src/components/search/Sort/Sort.tsx

```diff
@@ -25,23 +22,15 @@ function Sort() {
   } = useSearch()
 
   return (
-    <div className="sort / title-small">
-      <label htmlFor="select-sort">Sort by</label>
-      <UISelect
-        data-testid="search-sort"
-        onChange={(e) => setSort(keys[e.target.selectedIndex])}
-        value={sort}
-        aria-label="Product Sort"
-        id="select-sort"
-      >
-        {keys.map((key) => (
-          <option key={key} value={key}>
-            {OptionsMap[key]}
-          </option>
-        ))}
-      </UISelect>
-      <CaretDownIcon size={18} weight="bold" />
-    </div>
+    <Select
+      className="sort / title-small"
+      labelText="Sort by"
+      options={OptionsMap}
+      onChange={(e) => setSort(keys[e.target.selectedIndex])}
+      value={sort}
+      aria-label="Product Sort"
```

Comment:
I wonder if we can remove this `aria-label`. The `select`'s accessible name is already being set by its `label`, so there should be no need to use `aria-label` here. It's overriding something that is already working. Any thoughts @hellofanny?
🤔 

![CleanShot 2022-02-02 at 14 15 06](https://user-images.githubusercontent.com/381395/152203560-1d4f7122-f549-4f92-82fa-7ac287dde629.png)

Reply:
@filipewl the thing is that the label is optional. If we do not provide a text to it, then the aria label is required. I didn't want to overcomplicate things by making the attributes work like a "xor" (only one of them must be provided, but not both).

Since the aria label does not impact on anything 'visual' in the UI, I thought that making it always present regardless of label would maybe simplify things.

Reply:
I see, but in this case, we are already providing the `label` via `labelText="Sort by"` so I don't think we need provide `aria-label="Product Sort"`.

Reply:
I agree with with Filipe! Because we're duplicating unnecessary information. So it might be confusing.

Reply:
Oh, now I got it. You're not talking about the `Select` itself, but the implementation done by `Sort`, right? So if I got it right, this issue would be solved by simply not passing an `aria-label` to this implementation of `Select`?

We'll leave the responsibility of always including a `label` XOR an `aria-label` on the user's account? If we want to, I believe we can enforce this logic by using [discriminated union types](https://thoughtbot.com/blog/the-case-for-discriminated-union-types-with-typescript), but not sure if the added complexity is worth it?

Reply:
Yeah, I think we should remove the prop `aria-label="Product Sort"` in this `Sort.tsx` file.

> We'll leave the responsibility of always including a label XOR an aria-label on the user's account? If we want to, I believe we can enforce this logic by using [discriminated union types](https://thoughtbot.com/blog/the-case-for-discriminated-union-types-with-typescript), but not sure if the added complexity is worth it?

I think it's fine leaving that choice to the user. The logic to enforce the XOR behavior seems like it would overcomplicate `components/ui/Select/Select.tsx`'s implementation.

Reply:
Done at https://github.com/vtex-sites/base.store/pull/299/commits/25065dd8c0c7b682e550839edef64c5af6d21f63